docs/relevancyinterfaces.rst

```diff
@@ -0,0 +1,44 @@
+Relevancy Interfaces
+--------------------
+
+
+RankRelevancy
++++++++++++++
+
+This interface defines one method: ``rank``.  The ``rank`` method
+takes examples of relevant and not-relevant example descriptor vectors
+as :class:`numpy.ndarray` sequences and uses them to compute relevancy
+scores (on a ``[0, 1]`` scale) on a provided pool of other descriptor
+vectors.
+
+.. autoclass:: smqtk_relevancy.interfaces.rank_relevancy.RankRelevancy
+   :members:
+
+
+RankRelevancyWithFeedback
++++++++++++++++++++++++++
+
+This interface defines one method: ``rank_with_feedback``.  Like
+:meth:`RankRelevancy.rank`, ``rank_with_feedback`` takes examples of
+relevant and not-relevant example descriptor vectors as
+:class:`numpy.ndarray` sequences and uses them to compute relevancy
+scores (on a ``[0, 1]`` scale) on a provided pool of other descriptor
+vectors.  However, it also expects a sequence of corresponding UIDs
+for the pool vectors and additionally returns a sequence of UIDs,
+possibly not all from the pool, on which feedback would be most
+useful.
+
+.. autoclass:: smqtk_relevancy.interfaces.rank_relevancy.RankRelevancyWithFeedback
+   :members:
+   :private-members:
+
+
+RelevancyIndex
```

Comment:
I'm debating if we should add a deprecation notice here and refer to usage of `RankRelevancy`.

Reply:
what would be the correct way to signal this here?
simple adding a `[Deprecated]` under the header?

Reply:
I don't know a canonical "best" way, but I imagine we could add a narrative sentence as the first thing under this section so it is blatantly obvious (like make it bold or something).